Processing_data/Classes/Coordinate.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Processing_data.Classes
+{
+    /// <summary>
+    /// Represents a geographical coordinate
+    /// </summary>
+    public struct Coordinate
+    {
+        public double Latitude { get; set; }
```

Comment:
Avoid too much access. We don't need to change neither latitude nor longitude, so setter may be declared as `init`.

Reply:
thank you for new knowledge. i didn't know about `init`